README.md

```diff
@@ -1,43 +1,31 @@
 #Automated Image Cleanup for Amazon ECR
+
 The Python script and Lambda function described here help clean up images in [Amazon ECR](https://aws.amazon.com/ecr). The script looks for images that are not used in running [Amazon ECS](https://aws.amazon.com/ecs) tasks that can be deleted. You can configure the script to print the image list first to confirm deletions, specify a region, or specify a number of images to keep for potential rollbacks.
 
 ## Authenticate with AWS
 [Configuring the AWS Command Line Interface.](http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html)
 
-## Use virtualenv for Python execution
-
-To prevent any problems with your system Python version conflicting with the application, we recommend using virtualenv.
-
-Install Python:
-    `pip install python 3`
-
-Install virtualenv:
-
-    $ pip install virtualenv
-    $ virtualenv -p PATH_TO_YOUR_PYTHON_3 cloudformtion
-    $ virtualenv ~/.virtualenvs/cloudformtion
-    $ source ~/.virtualenvs/cloudformtion/bin/activate
-    
-## Generate the Lambda package
+## Generate and deploy the Lambda package
```

Comment:
Thanks for the additions and contributions. However, please add deployment steps with makefile as an optional step in readme(instead of replacing). We want to give our customers wide variety of deployment options. I will soon add ReadMe with steps for deployment using native aws cli calls, such as aws cloudformation package and aws cloudformation deploy.